### PR TITLE
Check for hydrogens in match and remove them to avoid indexing problem.

### LINKED ISSUE #67: Index error in min_symmetry RMSD method
The min_symmetry method gives an index error for some molecules when subsetting the ref_atoms matrix using `ref_atoms[match]` after the hydrogen atoms have been removed.

A suggested fix is to change:
`ref_atoms = ref.coords[ref.atom_dict['atomicnum'] != 1]`
to: 
`ref_atoms = ref.coords`

### PR DESCRIPTION
**Pull request by mwojcikowski**
This PR is related to #67

### CODE REVIEW

File: oddt/spatial.py [hunk 1]
```diff
@@ -150,20 +160,31 @@ def rmsd(ref, mol, ignore_h=True, method=None, normalize=False):
...
+                raise ValueError('Could not find any match between molecules.')
+            # calculate RMSD between all matches and retain the smallest
+            for match in matches:
+                match = np.array(match, dtype=int)
+                if is_openbabel_molecule(mol):
+                    match -= 1  # OB has 1-based indices
+                tmp_dict = mol.atom_dict[match]
+                mol_atoms = tmp_dict[tmp_dict['atomicnum'] != 1]['coords']
+                if mol_atoms.shape != ref_atoms.shape:
+                    raise ValueError('Molecular match got wrong number of '
```
Comment (marta-sd) [inline oddt/spatial.py]: is it possible to end up here?

File: oddt/spatial.py [hunk 2]
```diff
@@ -173,7 +194,8 @@ def rmsd(ref, mol, ignore_h=True, method=None, normalize=False):
             rmsd /= np.sqrt(mol.num_rotors)
```
Comment (marta-sd) [inline oddt/spatial.py]: we don't test this part

File: oddt/spatial.py [hunk 3]
```diff
@@ -150,20 +160,31 @@ def rmsd(ref, mol, ignore_h=True, method=None, normalize=False):
...
+                    min_rmsd = rmsd
+            return min_rmsd
     elif ignore_h:
-        hvy_map = np.argwhere(mol.atom_dict['atomicnum'] != 1).flatten()
-        mol_atoms = mol.coords[hvy_map]
-        ref_atoms = ref.coords[hvy_map]
+        mol_atoms = mol.coords[mol.atom_dict['atomicnum'] != 1]
+        ref_atoms = ref.coords[ref.atom_dict['atomicnum'] != 1]
     else:
         mol_atoms = mol.coords
```
Comment (marta-sd) [inline oddt/spatial.py]: we don't test this variant

Comment (mwojcikowski) [reply oddt/spatial.py]: This is too variable across different versions of OB/RDKit, so I didn't test it. What could go wrong there :)

**Pull request merged**